ui/fields/currency.php

```diff
@@ -44,7 +44,7 @@
             ?>]$/.test( $( this ).val() ) ) {
                 var newval = $( this )
                     .val()
-                    .replace( /[^0-9\<?php echo esc_js( $currency_sign ); ?>\<?php
+                    .replace( /[^0-9-\\<?php echo esc_js( $currency_sign ); ?>\<?php
```

Comment:
No, I tested it in js console as I had the same thought. It doesn't have a/0 before so not interpreted as a range character.